dts/bindings/spi/atmel,sam0-spi.yaml

```diff
@@ -27,12 +27,19 @@ properties:
       required: true
       description: Data Out Pinout
 
-    rxdma:
-      type: int
-      required: false
-      description: Receive DMA channel
-
-    txdma:
-      type: int
-      required: false
-      description: Transmit DMA channel
+    dmas:
+      description: |
+        Optional TX & RX dma specifiers.  Each specifier will have a phandle
+        reference to the dmac controller, the channel number, and peripheral
+        trigger source.
+
+        For example dmas for TX, RX on SERCOM3
+           dmas = <&dmac 0 0xb>, <&dmac 0 0xa>;
```

Comment:
The channel should be different on SPI.
dmas = <&dmac **1** 0xb>, <&dmac **0** 0xa>; maybe?

Reply:
Done